slides/week3/week3.org

```diff
@@ -0,0 +1,54 @@
+#+TITLE: Week 3
+#+AUTHOR: Kristaps Berzinch, Raymond Ortiz
+#+EMAIL: kristaps@robojackets.org, rortiz9@gatech.edu
+
+* TODO Needs speaker notes
+
+* What are we doing today?
+- Data structures and algorithms in C++
+- Brancing and merging in Git
+
+* What's a constructor?
+- A special type of function that instantiates a class
+- Basically, it sets up an object and returns it
+
+* What's a container?
+- An object you can use to store other objects
+- They can be ordered, associative, both, or neither
+  - *Ordered:* The contents are in a specific order
+  - *Associative:* Two objects are associated with each other within the container
+- TODO am I missing any important properties here? I want to say "uniqueness" or something...
+
+* Common containers
+| =array= | a static-length ordered list of objects |
+| =vector= | a dynamic-length ordered list of objects |
+| =set= | an ordered group of unique objects |
```

Comment:
I'll add to speaker notes.

RE unordered_containers: we can mention that each container (I think?) has an unordered version. We should teach unordered_set just to show how that works with iterators.